modules/ztp-deploying-siteconfig-hub-side-template.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-deploying-siteconfig-hub-side-template_{context}"]
+= Deploying a SiteConfig with hub-side templating
+
+The following is an example `SiteConfig` custom resource (CR) that uses the `spec.clusters.siteConfigMap.data` field to specify per-site data to be used in hub side templating.
```

Comment:
```suggestion
The following is a `SiteConfig` custom resource (CR) example that includes the new optional `spec.clusters.siteConfigMap`  where the `data` field contains per-site data to be used in hub side templating.
```

Reply:
There can be an issue with referring to a feature as "new" or "optional" in product documentation because, as time passes, the feature ceases to be new but the documentation won't be updated to reflect that. Release Notes are the best place to point out the newness of a feature.
I've rephrased though, in order to draw attention to the `spec.clusters.siteConfigMap` section and mention the `data` field separately

Reply:
Ok, but I think we should still say that it's an optional section, it's not mandatory. :thinking: